### ISSUE TITLE
US31 - Compartilhar pedidos

### ISSUE THREAD

**Issue by juniopereirab** [mia-ajuda/Documentation#33]
### Descrição:

Eu, como usuário que oferece ajuda, gostaria de compartilhar um pedido de ajuda em redes sociais, para encontrar outras pessoas que possam ajudar

### Critérios de aceitação:

- Link para compartilhar o pedido foi criado.
- Funcionalidade testada;

### Tarefas:

- [ ] Gerar um link do pedido que o usuário deseja compartilhar nas redes sociais.
- [ ] Testar funcionalidade

**Comment by AnaCarolcs** [comment]
@juniopereirab is this issue done ?

**Comment by juniopereirab** [comment]
> @juniopereirab is this issue done?

No, because the priority is defined as "WOULD", so this issue was available to someone that wanted to complete this. Since this issue don't aggregate value to the release.

**Comment by mangabeiras** [comment]
Testei hoje para ver se essa issue ja foi realizada. Ainda não foi executada, pois não existe forma d compartilhar o pedido de ajuda.

**Comment by AnaCarolcs** [comment]
I saw in the roadmap that this US should been done in Release 1. And it was not prioritize in Release 2.

**Comment by devsalula** [comment]
This is going to be in Release 2. Use deep link to redirect to the app in certain help request.
